sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -612,7 +624,10 @@ public void setValueDeserializer(String valueDeserializer) {
      * Coder)} to provide the value coder explicitly.
      */
     public Read<K, V> withValueDeserializer(Class<? extends Deserializer<V>> valueDeserializer) {
-      return toBuilder().setValueDeserializer(valueDeserializer).build();
+      return toBuilder()
+          .setValueDeserializer(valueDeserializer)
```

Comment:
The same question as for `withKeyDeserializer`

Reply:
I will remove it as mentioned above.